src/tribol/interface/mfem_tribol.cpp

```diff
@@ -369,7 +369,7 @@ mfem::ParGridFunction& getMfemPressure( IndexT cs_id )
   return cs->getMfemSubmeshData()->GetSubmeshPressure();
 }
 
-void updateMfemParallelDecomposition()
+void updateMfemParallelDecomposition( int n_ranks )
```

Comment:
It is common for this routine to be called cycle to cycle in an explicit code. Is there a disadvantage to potentially changing the number of ranks argument between cycle calls? Do we want to guard against this? Do we only want to decompose over a fixed set of ranks for a given coupling scheme as long as that coupling scheme persists?